catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/transformers/sierra/SierraCreators.scala

```diff
@@ -0,0 +1,39 @@
+package uk.ac.wellcome.transformer.transformers.sierra
+
+import uk.ac.wellcome.models.{AbstractAgent, Organisation, Person}
+import uk.ac.wellcome.transformer.source.{MarcSubfield, SierraBibData}
+
+trait SierraCreators extends MarcUtils {
+
+  def getCreators(bibData: SierraBibData): List[AbstractAgent] = {
```

Comment:
A bit late now, but could you write a comment (or just copy Silver’s!) that explains the intent of the transformation you’re doing.

It makes it easier to review whether the transformation is correct, and means it’s easier to see if we need to make new changes compared to sierratransformable.